metrics.yaml

```diff
@@ -0,0 +1,283 @@
+      - rs01-event
+    description: |
+      The largest depth reach on the page, as a
+      proportion of the total page height.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  max_pixel_scroll_depth:
+    type: quantity
+    unit: pixels
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The largest scroll pixel depth reached on the page.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  scroll_height:
+    type: quantity
+    unit: pixels
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      The total scroll height of the page, taken
+      from `document.documentElement.scrollHeight` at
+      the same interval as the other scroll fields.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+page.visit:
+  start:
+    type: datetime
+    time_unit: millisecond
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      When did the page visit start.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  stop:
+    type: datetime
+    time_unit: millisecond
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      When did the page visit stop.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+event:
+  start:
+    type: datetime
+    time_unit: millisecond
+    lifetime: ping
+    send_in_pings:
+      - rs01-event
+    description: |
+      Noting when the event started. If sent in a ping with
+      reason `attention`, this field notes when an inactive tab with
+      a page loaded in it has been given active focus
+      or a new page loads in an already-active tab.
+      If otherwise sent in a ping with reason `audio`, this
+      field notes when an unmuted audio element began
+      playing in the active tab.
+    bugs:
+      - https://github.com/mozilla-rally/rally-study-01/issues/106
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1703279#c2
+    notification_emails:
+      - hulmer@mozilla.com
+    expires: never
+
+  stop:
+    type: datetime
```

Comment:
Thanks for this ... I am ALL FOR this being a `datetime` over the timestamp we currently have for obvious reasons you're familiar with :)

Reply:
Eheheh :-D Yeah... "data is the new oil...". NO, NOT QUITE. Data is a new pain :-)